mp/game/momentum/resource/momentum_english_ref.res

```diff
@@ -397,7 +397,7 @@
         "MOM_NotApplicable" "N/A"
 
         // WEB/API/SERVER RESPONSES
-        "MOM_API_WaitingForResponse" "Waiting for API response..."
+        "MOM_API_WaitingForResponse" "Getting response..."
```

Comment:
This token was shared with other panels, and "Getting response" isn't much better, in my opinion. As a user seeing this, what is it "getting" a "response" of?

If you just wanted it shorter try "Fetching data..." instead, though, this token is definitely shared among other panels, so I don't think changing this is a good idea without seeing how it affects everything else.